src/main/app/claims/models/partyStatement.ts

```diff
@@ -0,0 +1,24 @@
+
+import { Serializable } from 'models/serializable'
+import { Offer } from 'claims/models/offer'
+
+export class PartyStatement implements Serializable<PartyStatement> {
+  type?: string
+  madeBy?: string
+  offer?: Offer
```

Comment:
I my previous comment I meant that only `offer` is an optional field. Other two should be mandatory.